### PR TITLE
Keras tutorial translate to bahasa indonesia (id)

### PR DESCRIPTION
**Pull request by csahat**
@lamberta sorry for the inconvenience, this is my new pull request for translation of keras tutorials to Indonesian language.

### CODE REVIEW

**Comment by rendrat** [comment]
Hi @csahat, there are a few files being changes, and some files are conflicting, maybe you can pull the files again and commit it.

**Comment by csahat** [comment]
@rendrat I have resolve the conflict, can you check it again? Thank you

**Comment by rendrat** [comment]
can we do like for one pull request for one file notebook ? cause it takes time to review one pull request with more then one file notebook.
Thanks

File: site/id/README.md [hunk 1]
```diff
@@ -7,17 +7,6 @@ Jika Anda memiliki saran untuk meningkatkan terjemahan ini, silakan kirim pull r
 Untuk menjadi sukarelawan untuk menulis atau memeriksa terjemahan komunitas, hubungi
 [daftar docs@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
 
-## Community translations
```
[the remark this answers is not in the document]
Comment (csahat) [reply site/id/README.md]: @rendrat If I am not wrong, @febriaroo said that the sentences that are still in English in readme.md should be removed. CMIIW

**Comment by csahat** [comment]
> can we do like for one pull request for one file notebook ? cause it takes time to review one pull request with more then one file notebook. I don't know, maybe @lamberta and @febriaroo have another opinion about this pull request mechanism ? please CMIIW.
> Thanks

Should I make new pull requests for every notebook then?

**Comment by lamberta** [comment]
Pull requests should be broken up by what's reasonable for a single reviewer to read through.

If making a bunch of typo fixes across many files, that's easy enough to look over all the diffs.

But despite how great these notebooks are 😀 you'll have a difficult time finding a reviewer to read through them all in one sitting---which means they'll start, stop, start, etc. and your PR will get held up (if finished, at all).

It's like chunks of work and as the community grows, it's more efficient to have separate PRs going with multiple reviewers working at the same time. (Of course, for a small community it's usually the same people doing the work, but best to get in the habit now to encourage more contributors.)

**Comment by csahat** [comment]
@lamberta okay, I will break it into several PR. Thank you

**Pull request closed without merging**